patterns.py

```diff
@@ -62,4 +62,7 @@ def railgun(light):
             light.neopixel[pixel] = light.primary
         else:
             light.neopixel[pixel] = (0, 0, 0)
-    light.neopixel.show()
\ No newline at end of file
+    light.neopixel.show()
```

Comment:
⭐ ⭐ could you do the `(loopcount - light.pattern_starting_loop) % 16` above since it's more for this pattern only?